lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1391,7 +1299,7 @@ public function getAssociationMappings()
      *
      * @return string The column alias.
      */
-    public function getFieldName($columnName)
+    public function getFieldName(string $columnName)
```

Comment:
`@param` can be removed. `@return` too, after return type was added.

Maybe other methods are similar in this class.